src/style.css

```diff
@@ -1,3 +1,163 @@
 body {
   margin: 0;
+  --font-family: "Roboto", sans-serif;
+  --active-star: url("./images/star-active.svg");
+  --empty-star: url("./images/star.svg");
+  --button-color: #00acdc;
+  --main-color: #060b35;
 }
+
+h2,
+h3,
+h4,
+p {
+  margin: 0;
+}
+
+/* #region stars */
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(./images/star.svg);
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  margin-right: 4px;
+  background-position: center;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.stars--5 .stars__star:nth-child(-n + 5),
+.stars--4 .stars__star:nth-child(-n + 4),
+.stars--3 .stars__star:nth-child(-n + 3),
+.stars--2 .stars__star:nth-child(-n + 2),
+.stars--1 .stars__star:nth-child(-n + 1),
+.stars--0 .stars__star:nth-child(-n) {
+  background-image: url(./images/star-active.svg);
+}
+/* #endregion */
+
+/* #region card */
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  min-height: 406px;
+  box-sizing: border-box;
+
+  padding: 32px 16px 16px;
+  border-radius: 5px;
+  left: 0.5%;
```

Comment:
Why you add this 4 rules below (left, right, top, bottom) if they don't work

Reply:
Sorry, will change and will be aware of checking before adding rules like these next time